ui/lib/src/chat/discussion.ts

```diff
@@ -108,7 +108,12 @@ function prependChatInput(chatInput: HTMLInputElement, prefix: string): void {
 let mouchListener: EventListener;
 
 const setupHooks = (ctrl: ChatCtrl, chatEl: HTMLInputElement) => {
-  const storage = tempStorage.make('chat.input');
+  const oppId = ctrl.opts.data.opponent?.user?.id ?? 'anon';
+  const key = `chat.input.${oppId}`;
+  Object.keys(sessionStorage)
```

Comment:
Not a fan that instanciating a chat widget causes the traversal of all session storage objects.